src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2330 @@ const ActorInit En_Ossan_InitVars = {
+            func_8010B720(globalCtx, this->unk_200[this->unk_252]->actor.textId);
+            func_80078884(0x4809);
+        }
+    }
+}
+
+void func_80AC4D20(EnOssan* this, GlobalContext* globalCtx, Player* player) {
+    Math_SmoothScaleMaxF(&this->unk_2D4, 0.0f, 0.5f, 10.0f);
+    if ((this->unk_2D4 < 0.5f) && (this->unk_2D4 > -0.5f)) {
+        func_80AC3350(this, globalCtx, 0.0f);
+    }
+    func_80AC3350(this, globalCtx, this->unk_2D4);
+    if (this->unk_2D4 == 0.0f) {
+        func_80AC39AC(globalCtx, this);
+    }
+}
+
+void func_80AC4DDC(EnOssan* this, GlobalContext* globalCtx, Player* player) {
+    if (!func_80AC652C(this)) {
+        osSyncPrintf("%s[%d]:" VT_FGCOL(GREEN) "ズーム中！！" VT_RST "\n", "../z_en_oB1.c", 2355);
+        return;
+    }
+    Math_SmoothScaleMaxF(&this->unk_2D4, 0.0f, 0.5f, 10.0f);
+    if ((this->unk_2D4 < 0.5f) && (this->unk_2D4 > -0.5f)) {
+        func_80AC3350(this, globalCtx, 0.0f);
+    }
+    func_80AC3350(this, globalCtx, this->unk_2D4);
+    if (this->unk_2D4 == 0.0f) {
+        func_8010B720(globalCtx, 0x3010);
+        this->unk_1FC = 0x14;
+    }
+}
+
+void func_80AC4EC8(GlobalContext* globalCtx, EnOssan* this) {
+    Player* player = PLAYER;
+
+    // Obtained for the first time!
+    osSyncPrintf("\n" VT_FGCOL(YELLOW) "初めて手にいれた！！" VT_RST "\n\n");
+    func_8002F434(&this->actor, globalCtx, this->unk_200[this->unk_252]->unk_1A4, 120.0f, 120.0f);
+    globalCtx->msgCtx.msgMode = 0x36;
+    globalCtx->msgCtx.unk_E3E7 = 4;
+    player->stateFlags2 &= ~0x20000000;
+    func_800BC490(globalCtx, 1);
+    Interface_ChangeAlpha(0x32);
+    this->unk_251 = 0;
+    func_80AC3350(this, globalCtx, 0.0f);
+    this->unk_1FC = 0xF;
+    // Start item pickup
+    osSyncPrintf(VT_FGCOL(YELLOW) "持ち上げ開始！！" VT_RST "\n\n");
+}
+
+void func_80AC4FAC(GlobalContext* globalCtx, EnOssan* this, u32 arg2) {
+    u32* new_var = &arg2;
+    u32 new_var2 = *new_var;
+
+    func_8010B720(globalCtx, new_var2);
+    this->unk_1FC = 0xE;
+}
+
+void func_80AC4FE0(GlobalContext* globalCtx, EnOssan* this, u16 arg2) {
+    func_8010B720(globalCtx, arg2);
+    this->unk_1FC = 0x17;
+}
+
+void func_80AC5014(GlobalContext* globalCtx, EnOssan* this) {
+    EnGirlA* girlA = this->unk_200[this->unk_252];
+
+    switch (girlA->unk_1BC(globalCtx, girlA)) {
+        case 0:
+            if ((girlA->actor.params == 12) && (gSaveContext.infTable[7] & 0x40)) {
+                func_80AC62C4(globalCtx, this);
+            } else {
+                func_80AC4EC8(globalCtx, this);
+                this->unk_251 = 0;
+                this->unk_2D0 = 0.0f;
+                girlA->unk_1AC(globalCtx, girlA);
+            }
+            break;
+        case 1:
+            girlA->unk_1C0(globalCtx, girlA);
+            func_80AC4FE0(globalCtx, this, 0x84);
+            this->unk_251 = 0;
+            this->unk_2D0 = 0.0f;
+            girlA->unk_1AC(globalCtx, girlA);
+            break;
+        case 2:
+            func_80078884(0x4806);
```

Comment:
```suggestion
            func_80078884(NA_SE_SY_ERROR);
```
Use the sfxId in sfx.h for the rest of func_80078884